docs/_data/menu-en.yml

```diff
@@ -2,37 +2,49 @@
   - title: Overview
     buttonTitle: Overview
     subfiles:
-      - title: Introduction
+      - title: Overview
         url: /en/Overview.html
-      - title: Dora
-        url: /en/overview/Why-Dora.html
-      - title: Getting Started
-        url: /en/overview/Getting-Started.html
+
+  - title: What is Alluxio
+    buttonTitle: What_is_Alluxio
+    subfiles:
+      - title: Introduction
+        url: /en/introduction/Introduction.html
+      - title: DORA Architecture
+        url: /en/introduction/Dora-Architecture.html
+      - title: Glossary
+        url: /en/introduction/Glossary.html
 
   - title: Core Services
     buttonTitle: Core_Services
     subfiles:
-      - title: Data Caching
+      - title: Caching
```

Comment:
`Data Caching` makes it more clear that this doc is specific to caching data vs the other metadata cache page. otherwise it may seem like metadata caching is a subset of the caching doc